test-case/check-pause-release-suspend-resume.sh

```diff
@@ -51,45 +51,39 @@ set -e
 ##  * No unexpected errors should be present in dmesg during or after test
 ##      completion.
 
+# shellcheck source=case-lib/lib.sh
 source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
 
-OPT_NAME['m']='mode'         OPT_DESC['m']='test mode. Example: playback; capture'
-OPT_HAS_ARG['m']=1             OPT_VAL['m']='playback'
+CASEDIR=$(dirname "${BASH_SOURCE[0]}")
 
-OPT_NAME['p']='pcm'          OPT_DESC['p']='audio pcm. Example: hw:0,0'
-OPT_HAS_ARG['p']=1             OPT_VAL['p']='hw:0,0'
+OPT_NAME['m']='mode'             OPT_DESC['m']='test mode. Example: playback; capture'
+OPT_HAS_ARG['m']=1               OPT_VAL['m']='playback'
```

Comment:
Please remove all these whitespace changes in this area. They make it impossible to find the real changes.
https://wiki.openstack.org/wiki/GitCommitMessages#Things_to_avoid_when_creating_commits